webpack/assets/javascripts/react_app/components/common/ModalProgressBar/index.js

```diff
@@ -0,0 +1,4 @@
+import ModalProgressBar from './ModalProgressBar';
+import './ModalProgressBar.scss';
```

Comment:
Could you move the `ModalProgressBar.scss` import to `ModalProgressBar.js`?
Then this file can be:
```js
export { default } from './ModalProgressBar';
```

Reply:
yes, thanks